libraries/engage/locations/components/FulfillmentPath/FulfillmentPath.jsx

```diff
@@ -0,0 +1,70 @@
+// @flow
+import * as React from 'react';
+import { Availability } from '@shopgate/engage/product';
+import { RadioGroup } from '../../../components';
+import { useFulfillmentState } from '../../locations.hooks';
+import {
+  PRODUCT_FULFILLMENT_METHOD_DIRECT_SHIP,
+  PRODUCT_FULFILLMENT_METHOD_IN_STORE_PICKUP,
+} from '../../constants';
+import { StockInfo } from '../StockInfo';
+import { container, pickUpContainer, radioGroup } from './FulfillmentPath.style';
+import { FulfillmentPathItem } from './FulfillmentPathItem';
+
+const directShip = 'product.fulfillment_selector.direct_ship';
+const pickUp = 'product.fulfillment_selector.pick_up_in_store';
+
+/**
+ * Renders the fulfillment path selector stage.
+ * @returns {JSX}
+ */
+export function FulfillmentPath() {
+  const { product, meta, changeFulfillment } = useFulfillmentState();
+  const cartItem = (meta && meta.cartItem) ? meta.cartItem : undefined;
+  const isPickUp = !!(cartItem
+    && (cartItem.fulfillment !== null && cartItem.fulfillment.method !== 'DIRECT_SHIP'));
```

Comment:
There is a const available for `DIRECT_SHIP`

Reply:
Sorry, no. The constant you are referring to has the value of `directShip` ... I can't use that one. And I don't wanna add another constant.